packages/storage-plus/src/keys.rs

```diff
@@ -196,6 +209,7 @@ impl<'a> Prefixer<'a> for Addr {
 impl<'a, T: Endian + Clone> PrimaryKey<'a> for IntKey<T> {
     type Prefix = ();
     type SubPrefix = ();
+    type NoPrefix = ();
```

Comment:
This is always `()`. Why do we need such a type?

Couldn't you just use `()` in the code, or remove the argument?

Reply:
Sure. I was following the existing impl, but yes, I don't think so much boilerplate is needed. Will simplify.

Reply:
Simplified `no_prefix`.

I thought `NoPrefix` was needed for traits matching, but it turns out it's superfluous.

This means we can also remove the associated `EmptyTrait`.